### PR TITLE
[Profiler] wording

### PR DESCRIPTION
**Pull request by noniagriconomie**
| Q             | A
| ------------- | ---
| Branch?       | master 
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | none
| License       | MIT
| Doc PR        | none

Small wording improvement

### CODE REVIEW

**Review by ro0NL — APPROVED** [review]

**Review by yceruto — APPROVED** [review]

**Comment by fabpot** [comment]
I don't see anything wrong in the current naming.

**Comment by ro0NL** [comment]
https://github.com/symfony/symfony/blob/76c98fda989a812a5265d9d3c97f855430821e29/src/Symfony/Component/HttpKernel/Profiler/FileProfilerStorage.php#L15 :grin: 

if we consider a "database" a more sophisticated system evolving around data, id say the implementation indeed relies on a storage system, not a database system.

**Comment by 94noni** [comment]
@fabpot just to name thing as they are :)

**Comment by javiereguiluz** [comment]
I agree that "database" looks wrong because we don't have a database here ... but what about removing any details about the storage?

Original:

```
No profiles found in the database.
Token ~ token|default('') ~ was not found in the database.
```

This PR:

```
No profiles found in the storage.
Token ~ token|default('') ~ was not found in the storage.
```

Proposal:

```
No profiles found.
Token ~ token|default('') ~ was not found.
```

**Comment by noniagriconomie** [comment]
We can yes 👍🏻

The main idea behind was to dissociate with the database word, as a newcomer may search for nothing the profiler table data inside its database, for example like the messenger table, but for nothing :)

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/info.html.twig [hunk 1]
```diff
@@ -4,7 +4,7 @@
     'no_token' : {
         status:  'error',
         title:   (token|default('') == 'latest') ? 'There are no profiles' : 'Token not found',
-        message: (token|default('') == 'latest') ? 'No profiles found in the database.' : 'Token "' ~ token|default('') ~ '" was not found in the database.'
+        message: (token|default('') == 'latest') ? 'No profiles found.' : 'Token "' ~ token|default('') ~ '" was not found.'
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/info.html.twig]: let's remove "was"

Comment (noniagriconomie) [reply src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/info.html.twig]: done

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @noniagriconomie.

**Pull request merged**